include/emp/math/Random.hpp

```diff
@@ -105,7 +105,14 @@ namespace emp {
     /// @return A pseudo-random double in the provided range.
     inline double GetDouble(const Range<double> range) noexcept {
       return GetDouble(range.GetLower(), range.GetUpper());
-     }
+    }
+
+    /// @return A pseudo-random double value between (0.0, 1.0]
+    inline double GetDoubleNonZero() noexcept {
+      double d = Get() / (double) RAND_CAP;
```

Comment:
Maybe delegate to existing implementation by calling GetDouble({std::numeric_limits::min(), 1.0}) ?

https://en.cppreference.com/w/cpp/types/numeric_limits/min